tinyDA/link.py

```diff
@@ -14,7 +14,7 @@ class Link:
         The model output
     likelihood : float
         The log-likelihood of the data, given the parameters.
-    qoi
+    qoi : numpy.ndarray
```

Comment:
`qoi` can be of `any` type.

Reply:
Ah, I see. I didn't get that

Reply:
I set it to any in the recent commit, hope that is what you meant